### ISSUE TITLE
Add cy.scrollIntoView()

### ISSUE THREAD

**Issue by jennifer-shehane** [cypress-io/cypress#498]
Add `cy.scrollIntoView()` command to scroll the element found in the previous command into view.

Addresses #313 

Examples:
```js
// last ul will be scrolled into view (to top/leftmost corner)
cy.get("ul").last().scrollIntoView()

// last ul will be scrolled into view - 50px on top
cy.get("ul").last().scrollIntoView({offset: "-50px"})

// last ul will be scrolled into view + 50px on top & 50px on bottom
cy.get("ul").last().scrollIntoView({offset: {top: "50px", left: "50px";}})

//  last ul will be scrolled into view over 200ms (default is 0 ms??)
cy.get("ul").last().scrollIntoView({duration: 200})

//  last ul will be scrolled to it's center (of it's height)
cy.get("ul").last().scrollIntoView({align: "50%"})

//  last ul will be scrolled to it's very bottom
cy.get("ul").last().scrollIntoView({align: {top: "100%", left: "100%"}})
```

**Comment by RandallKent** [comment]
The code for this is done, but this has yet to be released. We'll update the issue and reference the changelog when it's released.

**Comment by brian-mann** [comment]
Fixed in [`0.20.0`](https://docs.cypress.io/guides/references/changelog.html#0-20-0)
